README.md

```diff
@@ -7,6 +7,56 @@
 
 The purpose of this GitHub App is to serve the needs of the Git for Windows project, implementing all kinds of useful automation.
 
+The App itself is implemented as an Azure Function that performs quick tasks itself and hands off more complex tasks to GitHub workflows in the [`git-for-windows-automation` repository](https://github.com/git-for-windows/git-for-windows-automation).
+
+## Slash commands
+
+The GitForWindowsHelper GitHub App supports so-called "slash commands", i.e. commands that start with a forward slash and that are issued via comments in GitHub Issues or Pull Requests.
+
+### `/hi`
+
+**Where can it be called?** In issues/PRs of any repository in which the GitForWindowsHelper GitHub App is installed.
+
+**What does it do?** The app responds in a separate comment, saying "Hi @&lt;login&gt;" where `<login>` is your GitHub login name.
+
+### `/add release note <type> <message>`
```

Comment:
Do we also want to document the alternate spellings (`/add relnote`) and the invocation without `<type>`/`<message>`?